src/ResultsItem.tsx

```diff
@@ -14,71 +13,51 @@ interface Props<T> {
   onMouseLeave?: (e: any /* Event */) => void;
   // onClick callback
   onClickItem?: (e: any /* Event */) => void;
-  // set to true to highlight the given item
-  highlighted?: boolean;
+  // set to true if the item is currently selected
+  selected?: boolean;
   // optional style override
   style?: React.CSSProperties;
 }
 
 interface State {
   // set to true to highlight
-  hover: boolean;
+  highlighted: boolean;
 }
 
-const ITEM_STYLE: React.CSSProperties = {
-  height: DEFAULT_HEIGHT,
-  lineHeight: `${DEFAULT_HEIGHT}px`,
-  fontSize: 24,
-  borderStyle: 'solid',
-  borderColor: COLORS.DARKGRAY,
-  borderTopWidth: 0,
-  borderLeftWidth: 1,
-  borderRightWidth: 1,
-  borderBottomWidth: 1,
-  boxSizing: 'border-box',
-};
-
-const ITEM_HOVER_STYLE: React.CSSProperties = {
-  backgroundColor: COLORS.GRAY,
-};
-
 export default class ResultRenderer<T> extends React.PureComponent<
   Props<T>,
   State
 > {
   static defaultProps = {
-    highlighted: false,
+    selected: false,
   };
 
   state: State = {
-    hover: false,
+    highlighted: false,
```

Comment:
I would name this accordingly to its type.
Since it is a boolean, call this `isHighlighted`(just like how we named it when we pass it down)

Reply:
👍